src/components/TableOfContents/TableOfContentsMobile.tsx

```diff
@@ -21,9 +20,6 @@ export interface IPropsTableOfContentsMobile {
 }
 
 const Mobile: React.FC<IPropsTableOfContentsMobile> = ({ items, maxDepth }) => {
-  // TODO: Replace with direct token implementation after UI migration is completed
-  const lgBp = useToken("breakpoints", "lg")
```

Comment:
Related to the previous comment about doing these changes here. I wouldn't do them here.